src/main/java/seedu/address/logic/parser/FindContactCommandParser.java

```diff
@@ -0,0 +1,67 @@
+package seedu.address.logic.parser;
+
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_MODULE;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_NAME;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Scanner;
+
+import seedu.address.logic.commands.FindContactCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+import seedu.address.model.person.ModuleTakenPredicate;
+import seedu.address.model.person.NameContainsKeywordsPredicate;
+import seedu.address.model.tag.NusModule;
+
+/**
+ * Parses input arguments and creates a new FindContactCommand object
+ */
+public class FindContactCommandParser implements Parser<FindContactCommand> {
+
+    /**
+     * Parses the given {@code String} of arguments in the context of the FindContactCommand
+     * and returns a FindContactCommand object for execution.
+     * @throws ParseException if the user input does not conform the expected format
+     */
+    public FindContactCommand parse(String args) throws ParseException {
+        String trimmedArgs = args.trim();
+
+        if (trimmedArgs.isEmpty()) {
+            throw new ParseException(
+                    String.format(MESSAGE_INVALID_COMMAND_FORMAT, FindContactCommand.MESSAGE_USAGE));
+        }
+
+        // John Lim Jacky
+        // n/ Johnathan Sally
+        // m/ Jacob
```

Comment:
As far as I can tell, the AB3 command syntax (which we're inheriting) does not have a space in between the argument prefixes and contents. (e.g. `add n/John Doe` instead of `add n/ John Doe`.)

Maybe use the parsing utilities instead? (See the AddCommandParser / AddContactCommandParser)

Reply:
I had some issues when trying to implement using the existing parse utilities. Will try to change my code to use it later on!